packages/common/src/fixtures/select/1-static-selects.xml

```diff
@@ -13,10 +13,10 @@
       <itext>
         <translation lang="French">
           <text id="fruit:label">
-            <value>1. Sélectionnez un fruit</value>
+            <value>`1. Sélectionnez un fruit`</value>
```

Comment:
These had to be changed because without the backtick they were rendering as an ordered list which broke a test.